stable/spinnaker/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.12.0
+version: 1.12.1
```

Comment:
This is not a big fix please bump to 1.13.0

Reply:
Would you say it's a minor update? I ask only because currently, when we use the Spinnaker Helm Chart, it cannot create the Minio microservice because the image it tries to pull doesn't exist anymore. We aren't adding functionality, instead we are just fixing what currently isn't done (puling of image).

Reply:
Humm ok sounds good